packages/discovery-provider/src/queries/get_premium_track_signatures.py

```diff
@@ -499,65 +496,6 @@ def get_nft_gated_premium_track_signatures(
         return result
 
 
-# Generates a premium content signature for each of the premium tracks.
-# Return a map of premium track id -> premium content signature.
-def get_premium_track_signatures(user_id: int, track_ids: List[int]):
```

Comment:
was this dead code I guess?

Reply:
it was. i was too eager in the beginning when working with the signatures. was pretty sure this was going to be used, similar to how nft gated tracks are batch-requested. never happened.